.github/ISSUE_TEMPLATE/experimental_feature.md

```diff
@@ -0,0 +1,28 @@
+---
+name: Experimental Feature Requirements
+about: Requirements for experimental features
+---
+
+# Experimental Feature Requirements
+
+This page lists the requirements for experimental features. Experimental features allow Istio work groups to collect more information
+on potential new functionality. This template should be completed users try out any new experimental features. Please check off and 
+document the steps as they are completed.
+
+**Feature Name:** 
+
+--- 
+
+### Requirements: 
+
+- [ ] Feature is reviewed in a work group meeting
+- [ ] RFC has been written
+- [ ] Feature is disabled by default
+- [ ] Steps for enabling the feature have been documented. This may include
+	instructions for building, running an `istioctl experimental command`, or
+	using the preview profile. 
+	**Link to instructions:** 
+- [ ] Plan exists for getting feedback. This may include user feedback meetings,
+	discuss.istio.io conversations, GitHub issues, or mailing lists. 
+> Plan for feedback
+- [ ] No impact on performance when the feature is disabled.
```

Comment:
nit: I think this line is part of the quite without an additional newline? or github syntax highlight is wrong

Reply:
Hmmm. It's showing up correctly in the preview, just not the syntax highlighting here.

Reply:
![image](https://user-images.githubusercontent.com/8484260/93375132-b311fb80-f825-11ea-8765-f41f07fda354.png)

Do the quotes make more sense than code blocks for text boxes to fill in?

Reply:
GitHub syntax highlighting wanted a newline between paragraph markers and bullets. Doesn't seem to have any effect on formatting.